pkg/filter/dt/filter_mysql_query.go

```diff
@@ -86,17 +91,19 @@ func (f *_mysqlFilter) processBeforeQueryUpdate(ctx context.Context, conn *drive
 
 func (f *_mysqlFilter) processAfterQueryDelete(ctx context.Context, conn *driver.BackendConnection, deleteStmt *ast.DeleteStmt) error {
 	has, xid := misc.HasXIDHint(deleteStmt.TableHints)
+	newCtx, span := tracing.GetTraceSpan(ctx, "mysql_process_after_query_delete")
```

Comment:
I recommend not adding TraceContext for this location. instead, add inside of func `executor.BeforeImage`:
```
newCtx, span := tracing.GetTraceSpan(ctx, "fetch_before_image")
```